.vscode/tasks.json

```diff
@@ -0,0 +1,15 @@
+{
+    // See https://go.microsoft.com/fwlink/?LinkId=733558 
+    // for the documentation about the tasks.json format
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "type": "npm",
+            "script": "build",
+            "group": {
+                "kind": "build",
+                "isDefault": true
+            }
+        }
+    ]
+}
```

Comment:
What are these .vscode files for? Should they be gitignored?

Reply:
Not actually sure where those came from. I commented them out and ran the app with no problems. We could probably delete those